Sources/_SwiftSyntaxMacros/MacroSystem+Builtin.swift

```diff
@@ -374,31 +374,6 @@ struct FileIDMacro: ExpressionMacro {
   }
 }
 
-struct FileMacro: ExpressionMacro {
```

Comment:
@DougGregor Do we still need this?

Reply:
It can't really work until we get build configuration information into the macro evaluation context, so I'm fine with removing it.